app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -535,6 +536,7 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
 
     /**
      * @var \Magento\CatalogInventory\Model\ResourceModel\Stock\ItemFactory
+     * @deprecated
```

Comment:
A good practice is to have the description for each `deprecated` tag

Reply:
Done.